cypress/support/utils/apiUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { addCandidatePayload } from "../API_Helpers/payloads/addCandidatePayload";
+import { addCandidateResponse } from "../API_Helpers/responses/addCandidateResponse";
+
+
+declare global {
+    namespace Cypress {
+        interface Chainable {
```

Comment:
the login can be a command, and it's a must, right? you do the login in each scenario